packages/caver-klay/caver-klay-accounts/src/index.js

```diff
@@ -68,6 +68,70 @@ function coverInitialTxValue(tx) {
   return tx
 }
 
+function resolveArgsForSignTransaction(args) {
```

Comment:
Please add description of this function.

Reply:
Description for resolveArgsForSignTransaction is added